sdk/template-LLC/Azure.Template.LLC/src/Generated/TemplateServiceClient.cs

```diff
@@ -76,21 +83,28 @@ public virtual async Task<Response> NoRequestBodyResponseBodyAsync(int id, int?
             }
         }
 
-        /// <summary> No RequestBody and ResponseBody. </summary>
-        /// <param name="id"> Query parameter Id. </param>
-        /// <param name="top"> Query parameter top. </param>
-        /// <param name="skip"> Query parameter skip. </param>
-        /// <param name="status"> Query parameter status. </param>
+        /// <summary> Create or update resource. </summary>
+        /// <param name="resourceId"> The id of the resource. </param>
         /// <param name="context"> The request context. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="resourceId"/> is null. </exception>
+        /// <remarks>
+        /// Schema for <c>Response Body</c>:
+        /// <code>{
+        ///   name: string,
+        ///   id: string
+        /// }
+        /// </code>
+        /// 
+        /// </remarks>
 #pragma warning disable AZC0002
-        public virtual Response NoRequestBodyResponseBody(int id, int? top = null, int skip = 12, string status = "start", RequestContext context = null)
+        public virtual Response Create(string resourceId, RequestContext context = null)
```

Comment:
I think this should take a RequestContent - this would be typical for a Create operation, where the caller provides the details of the resource to create.